indecision-plaza/app/food-finder/food-finder.component.ts

```diff
@@ -29,29 +28,31 @@ export class FoodFinderComponent {
   }
 
   findFood() {
-    this.foodSvc.getNearbyFood(this.categoryMap[this.chosenCategory]).forEach((data) => {
-      // Grab the list of businesses from the JSON
-      let places = data.businesses
-
-      if(places.length !== 0) {
-        // Choose one random business
-        let chosenPlaceJSON = places[Math.floor(Math.random() * places.length)]
-        // Convert chosen business into an Entity that is read into the info card
-        this.chosenPlace = new Business(chosenPlaceJSON);
-        console.log(this.chosenPlace.toLongString())
-      }
-      else {
-        this.chosenPlace = Business.makeEmpty();
-        console.log("No results were found");
-      }
+    enableLocationRequest();
```

Comment:
Would it make more sense to move this to the finder service?

Reply:
This makes sense, but seems to have some asynchronous issues. We need this to occur on load of the page.